packages/react/src/card/card.tsx

```diff
@@ -42,7 +42,7 @@ export const CardBody = forwardRef<HTMLDivElement, CardBaseProps>(
     const Component = asChild ? Slot : "div";
     return (
       <Component {...rest} className={clsx("hds-card__body", className as undefined)} ref={ref}>
-        {children}
+        <div className="hds-card__centerbody">{children}</div>
```

Comment:
Use the `clsx` utility, even for single classnames that does not allow extension. It ensures type-safety. Meaning if the class `hds-card__centerbody` is ever removed/renamed, typescript will complain